### PR TITLE
ffdec: Add version 15.0.0

### LINKED ISSUE #7682: [Request] Add FFDec
### Information

**Name:** ffdec

**Description:** Open Source Flash SWF decompiler and editor.

**Homepage:** https://github.com/jindrapetrik/jpexs-decompiler

**Download link(s):** https://github.com/jindrapetrik/jpexs-decompiler/releases/latest

**Some indication of popularity/repute:** 3k stars, 508 forks, 181 watches on GitHub.

### PR DESCRIPTION
**Pull request by samiksome92**
Closes #7682

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/ffdec.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "15.0.0",
+    "description": "Open Source Flash SWF decompiler and editor.",
+    "homepage": "https://github.com/jindrapetrik/jpexs-decompiler",
+    "license": "GPL-3.0-only",
```
Comment (niheaven) [inline bucket/ffdec.json:5]:
```suggestion
    "license": "GPL-3.0-or-later",
```

File: bucket/ffdec.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "https://github.com/jindrapetrik/jpexs-decompiler/releases/download/version15.0.0/ffdec_15.0.0.zip",
+    "hash": "1f0592b5ee0e079beb19b486787a2b622e3f6a386b11902f832dd3be55c7aff4",
+    "shortcuts": [
```
Comment (niheaven) [inline bucket/ffdec.json:8]:
```suggestion
    "bin": "ffdec.jar",
    "shortcuts": [
```

File: bucket/ffdec.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        [
+            "ffdec.exe",
+            "FFDec"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/jindrapetrik/jpexs-decompiler/releases/latest",
+        "regex": "version(\\d+\\.\\d+\\.\\d+)"
```
Comment (niheaven) [inline bucket/ffdec.json:16]:
```suggestion
        "github": "https://github.com/jindrapetrik/jpexs-decompiler",
        "regex": "version([\\d.]+)"
```

**Review by niheaven — COMMENTED** [review]
config file is in `$APPDATA$\JPEXS\FFDec`, and I haven't find a way to make it portable

File: bucket/ffdec.json (hunk 2 above)
Comment (samiksome92) [reply bucket/ffdec.json:8]: Should we add ffdec.jar or ffdec.exe?

Comment (niheaven) [reply bucket/ffdec.json:8]: `ffdec.exe` don't has CLI, it is a pure GUI program.

Comment (samiksome92) [reply bucket/ffdec.json:9]: OK, thanks. I assumed it simply called `java -jar ffdec.jar <args>`

**Review by niheaven — APPROVED** [review]

**Pull request merged**